release/test_tag_release.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+source release_gold.sh --source-only
+
+function delete_tag {
+	local curl_response=$(\
+		curl "https://api.github.com/repos/${LIFERAY_RELEASE_USER_REPOSITORY}/${LIFERAY_RELEASE_REPOSITORY_NAME}/git/refs/tags/${LIFERAY_RELEASE_VERSION}" \
+			--header "Accept: application/vnd.github+json" \
+			--header "Authorization: Bearer ${LIFERAY_RELEASE_GITHUB_PAT}" \
+			--include \
+			--request DELETE \
+			--silent)
+
+	if [ $(echo "${curl_response}" | awk '/^HTTP/{print $2}') -ne 204 ]
+	then
+		echo "Failed to delete tag ${LIFERAY_RELEASE_VERSION}."
+
+		exit
+	fi
```

Comment:
@lucasmiranda0, based on the content of the method `set_up`, this test will always fail unless some info gets inserted in the env variables manipulated by the `set_up`, am I right?

If so, can you please try to find out a way to make the test pass by default instead of fail?

You can try to use the GitHub user `liferay-release` and also print some help to the developer if some env variable should be populated in the `set_up`. Can you give it a try?

Remember: the test should pass by default